diff --git a/drivers/staging/Makefile b/drivers/staging/Makefile
index c7f8d8d..7f0d508 100644
--- a/drivers/staging/Makefile
+++ b/drivers/staging/Makefile
@@ -9,6 +9,7 @@ obj-$(CONFIG_RTL8192E)		+= rtl8192e/
 obj-$(CONFIG_RTL8723BS)		+= rtl8723bs/
 obj-$(CONFIG_R8712U)		+= rtl8712/
 obj-$(CONFIG_R8188EU)		+= r8188eu/
+obj-$(CONFIG_88XXAU)            += rtl8812au/
 obj-$(CONFIG_RTS5208)		+= rts5208/
 obj-$(CONFIG_NETLOGIC_XLR_NET)	+= netlogic/
 obj-$(CONFIG_OCTEON_ETHERNET)	+= octeon/
